config/Shared/config_default-ci.mysql.php

```diff
@@ -9,7 +9,7 @@
 
 $config[PropelConstants::ZED_DB_ENGINE] = PropelConfig::DB_ENGINE_MYSQL;
 $config[PropelConstants::ZED_DB_USERNAME] = 'root';
-$config[PropelConstants::ZED_DB_PASSWORD] = '';
+$config[PropelConstants::ZED_DB_PASSWORD] = 'secret';
```

Comment:
do we really need this here? As I can see for public suite we haven't specified passwords in this way

Reply:
It is needed to properly configure DB in GA.

Reply:
As I understood it is used only in ci.